src/main/scripts/org/reaktivity/specification/nukleus/tcp/streams/server.sent.data.multiple.streams/server/target.rpt

```diff
@@ -0,0 +1,117 @@
+#
+# Copyright 2016-2017 The Reaktivity Project
+#
+# The Reaktivity Project licenses this file to you under the Apache License,
+# version 2.0 (the "License"); you may not use this file except in compliance
+# with the License. You may obtain a copy of the License at:
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+#
+
+property nuklei ${nuklei:directory("target/nukleus-itests").streamsCapacity(1024 * 1024, 64 * 1024)}
+
+property targetInput ${nuklei.streams("target", "tcp#any")}
+property targetOutputEst ${nuklei.streams("tcp", "target")}
+
+property newTargetOutputEstId1 ${nuklei:newStreamId()}
+property newTargetOutputEstId2 ${nuklei:newStreamId()}
+
+# Override to test for reset
+property secondStreamWriteReplyFrameType [0x40 0x00 0x00 0x02]
```

Comment:
I thought it better not to add a spec script specifically for an implementation detail. But I can certainly do so if you prefer.